config/camera-manager.yaml

```diff
@@ -1,7 +1,22 @@
 tests:
-    - name: CameraSettings
-      camera_id: 2
-      param_name: wb-mode
-      param_value: 0
-      sub_param_name: wb-temp
-      sub_param_values: 4000
+  - name: CameraSettings
+    camera_id: 2
+    param_name: wb-mode
+    param_is_range: false
```

Comment:
Since it's the default you can consider dropping it here.

Reply:
Hm, I wanted to make it explicit that we have two tests with and two tests without range params.